XMonad/Hooks/FadeWindows.hs

```diff
@@ -76,17 +76,20 @@ import           Graphics.X11.Xlib.Extras                (Event(..))
 -- >     , handleEventHook = fadeWindowsEventHook
 -- >     {- ... -}
 -- >
--- > myFadeHook = composeAll [isUnfocused --> transparency 0.2
--- >                         ,                opaque
+-- > myFadeHook = composeAll [                 opaque
+-- >                         , isUnfocused --> transparency 0.2
 -- >                         ]
 --
 -- The above is like FadeInactive with a fade value of 0.2.
 --
--- FadeHooks do not accumulate; instead, they compose from right to
--- left like 'ManageHook's, so the above example @myFadeHook@ will
--- render unfocused windows at 4/5 opacity and the focused window
--- as opaque.  The 'opaque' hook above is optional, by the way, as any
--- unmatched window will be opaque by default.
+-- 'FadeHook's do not accumulate; instead, they compose from right to
+-- left like 'ManageHook's, so in the above example @myFadeHook@ will
+-- render unfocused windows at 4/5 opacity and the focused window as
+-- opaque.  This means that, in particular, the order in the above
+-- example is important.
+--
+-- The 'opaque' hook above is optional, by the way, as any unmatched
+-- window will be opaque by default.
```

Comment:
I don't think this one is true, either:
https://github.com/xmonad/xmonad-contrib/blob/f9c3f4ecf62eab2747bb537e693f155a615eb8eb/XMonad/Hooks/FadeWindows.hs#L208-L212
That seems like without `opaque`, unmatched windows get 0.93.

Am I understanding this correctly, @geekosaur?

Reply:
I just tested this and you seem to be right; can't believe I've not noticed that during testing!